force-app/main/default/classes/AuroraSpikeController.cls

```diff
@@ -167,13 +167,49 @@ public with sharing class AuroraSpikeController {
         }
     }
     
+    public static string createAuroraProject(Site__c site){
```

Comment:
This is throwing a deployment error because it's set to return a string but doesn't.

Reply:
Updated this to return the response string